README.md

```diff
@@ -12,3 +12,35 @@ The following plugins are offered:
 	Allows to package and distribute Python modules during Maven builds
 
 
+## Usage
+
+
+To use the plugin, add the following in your `pom.xml`
+
+```xml
+
+<build>
+        <plugins>
+        ...
+            <plugin>
+                <groupId>io.sqooba</groupId>
+                <artifactId>maven-python-plugin</artifactId>
+                <version>1.1.2</version>
+                <executions>
+                    <execution>
+                        <goals>
+                            <goal>test</goal>
+                            <goal>package</goal>
+                            <goal>install</goal>
+                        </goals>
+                    </execution>
+                </executions>
+                  <configuration>
+                      <pythonbuild>bdist_wheels</pythonbuild> <!-- select your packaging approach -->
```

Comment:
Could you add this configuration option in the "Configuration and defaults" section instead of here? As this parameter is optional and the build would work without it I would keep the pom snippet as minimal as possible

Reply:
Removed pom.xml from README.md in 4ec80f06565ef4df7bbde1310116f925e03b8ca8

Reply:
Add parameter doc in 2c56b7e